editor/src/dev/load_project.clj

```diff
@@ -58,10 +59,8 @@
   [:setup-workspace
    :fetch-libraries
    :resource-sync
-   :make-project
    :list-resources
-   :make-nodes
-   :setup-project
+   :make-project
```

Comment:
`:setup-project` was also merged into `:load-nodes`. And we no longer need the `project` to `:list-resources`. However, we do need it to be set up before `:read-resources`, because editor plugins may have registered additional resource-types.